docker/docker-compose.yml

```diff
@@ -6,31 +6,47 @@ services:
     restart: always
     container_name: datamart-postgres
     environment:
-        - POSTGRES_PASSWORD=postgres
-        - POSTGRES_USER=postgres
-        - POSTGRES_DB=wikidata
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=wikidata
     volumes:
-        - postgres:/var/lib/postgresql/data
-        - ../dev-env/data/postgres/datamart.sql.gz:/docker-entrypoint-initdb.d/01-datamart.sql.gz
-        - ../dev-env/data/postgres:/backup
+      - postgres:/var/lib/postgresql/data
+      - ../dev-env/data/postgres/datamart.sql.gz:/docker-entrypoint-initdb.d/01-datamart.sql.gz
+      - ../dev-env/data/postgres:/backup
     expose:
-        - "5432"
+      - "5432"
     ports:
-        - "5433:5432"
+      - "5433:5432"
 
   backend:
     build:
-        context: ..
-        dockerfile: docker/Dockerfile
+      context: ..
+      dockerfile: docker/Dockerfile
     container_name: datamart-backend
     expose:
-        - "80"
+      - "80"
     ports:
-        - "14080:80"
+      - "14080:80"
+    network_mode: "host"
+
+  es01:
```

Comment:
All changes to docker-compose.yml should also be reflected in `dev-env/docker-compose.yml`, so we can use elasticsearch and test everything in development, not just when deployed on a server.